src/components/HighContrastSelector/HighContrastSelector.tsx

```diff
@@ -0,0 +1,22 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+
+import 'HighContrastSelector.css';
```

Comment:
Can you please replace this with a getClassNames function? We've been working towards using merge-styles for all components.

The Box component will have an example to follow. 

In future, if you generate the component with `npm run create:component` it'll create the files you need.

Reply:
You can do the media queries like this:
https://www.npmjs.com/package/@uifabric/merge-styles#media-queries

Reply:
It looks like you can also import the selector from fabric if you want:
```import { HighContrastSelector } from 'office-ui-fabric-react/lib/Styling';```

Reply:
Yeah. I've already been working my way through this after having tried out: npm run create:component (just to see what that did vs what i made from scratch). I hadn't realized that we had a working React-flavored way to handle media queries, so I was sticking with the CSS. I'll add the .styles stuff back in.